### PR TITLE
Add TeamsVirtualAppointmentsPolicy cmdlet docs

### PR DESCRIPTION
**Pull request by emmanuelrocha001**

### CODE REVIEW

File: teams/teams-ps/teams/Get-CsTeamsVirtualAppointmentsPolicy.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+external help file: Microsoft.Teams.Policy.Administration.Cmdlets.Core.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://learn.microsoft.com/powershell/module/skype/get-csteamsvirtualappointmentspolicy
+schema: 2.0.0
+ms.author: erocha
```
Comment (get-itips) [inline teams/teams-ps/teams/Get-CsTeamsVirtualAppointmentsPolicy.md:6]: please also add "author" metadata, that is your GitHub username

Comment (emmanuelrocha001) [reply teams/teams-ps/teams/Get-CsTeamsVirtualAppointmentsPolicy.md:6]: addressed.

File: teams/teams-ps/teams/Set-CsTeamsVirtualAppointmentsPolicy.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+## SYNTAX
+
+```
+Set-CsTeamsVirtualAppointmentsPolicy [-EnableSmsNotifications <Boolean>] [-Identity] <String> [-Force]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Updates any of the properties of an existing instance instance of TeamsVirtualAppointmentsPolicy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsVirtualAppointmentsPolicy.md:24]:
```suggestion
Updates any of the properties of an existing instance of TeamsVirtualAppointmentsPolicy.
```

File: teams/teams-ps/teams/Get-CsTeamsVirtualAppointmentsPolicy.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+manager: sonaggarwal
+author: emmanuelrocha001
+---
+
+# Get-CsTeamsVirtualAppointmentsPolicy
+
+## SYNOPSIS
+This cmdlet is used to fetch policy instances of TeamsVirtualAppointmentsPolicy.
+## SYNTAX
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsVirtualAppointmentsPolicy.md:15]:
```suggestion

## SYNTAX
```

File: teams/teams-ps/teams/Get-CsTeamsVirtualAppointmentsPolicy.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsVirtualAppointmentsPolicy.md:103]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: teams/teams-ps/teams/Grant-CsTeamsVirtualAppointmentsPolicy.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+### GrantToTenant
+```
+Grant-CsTeamsVirtualAppointmentsPolicy [[-PolicyName] <String>] [-Global] [-Force] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet applies an instance of the TeamsVirtualAppointmentsPolicy to users or groups in a tenant.
+
+Pass in the `Identity` of the policy instance in the `PolicyName` parameter and the user identifier in the `Identity` parameter or the group name in the `Group` parameter. One of either `Identity` or `Group` needs to be passed.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsVirtualAppointmentsPolicy.md:43]:
```suggestion
Passes in the `Identity` of the policy instance in the `PolicyName` parameter and the user identifier in the `Identity` parameter or the group name in the `Group` parameter. One of either `Identity` or `Group` needs to be passed.
```

File: teams/teams-ps/teams/Grant-CsTeamsVirtualAppointmentsPolicy.md [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+
+Assigns a given policy to the tenant.
+
+### Example 3
+```powershell
+PS C:\> Grant-CsTeamsVirtualAppointmentsPolicy -Global -PolicyName sms-enabled
+```
+
+Note: _Using $null in place of a policy name can be used to unassigned a policy instance._
+## PARAMETERS
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsVirtualAppointmentsPolicy.md:76]:
```suggestion

## PARAMETERS
```

File: teams/teams-ps/teams/Grant-CsTeamsVirtualAppointmentsPolicy.md [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PolicyName
+Specifies the name of the policy to be assigned. The PolicyName is the policy identity minus the policy scope ("tag:"), e.g. a policy that has an identity of "Tag:Enabled" has a PolicyName of "Enabled".
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsVirtualAppointmentsPolicy.md:145]:
```suggestion
Specifies the name of the policy to be assigned. The PolicyName is the policy identity minus the policy scope ("tag:"), for example, a policy that has an identity of "Tag:Enabled" has a PolicyName of "Enabled".
```

File: teams/teams-ps/teams/Grant-CsTeamsVirtualAppointmentsPolicy.md [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsVirtualAppointmentsPolicy.md:175]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: teams/teams-ps/teams/New-CsTeamsVirtualAppointmentsPolicy.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+### Example 1
+```powershell
+PS C:\> New-CsTeamsVirtualAppointmentsPolicy -Identity sms-enabled
+```
+```output
+Identity                 EnableSmsNotifications                                                                          
+--------                 ----------------------                                                                          
+Tag:sms-enabled                            True
+```
+Creates a new policy instance with the identity enable-sms. `EnableSmsNotifications` will default to true, as it was not specified.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsVirtualAppointmentsPolicy.md:37]:
```suggestion
Creates a new policy instance with the identity enable-sms. `EnableSmsNotifications` will default to true if it is not specified.
```

File: teams/teams-ps/teams/New-CsTeamsVirtualAppointmentsPolicy.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+### Example 2
+```powershell
+PS C:\> New-CsTeamsVirtualAppointmentsPolicy -Identity disable-sms -EnableSmsNotifications $false
+```
+```output
+Identity                 EnableSmsNotifications                                                                          
+--------                 ----------------------                                                                          
+Tag:sms-enabled                            False
+```
+Creates a new policy instance with the identity sms-disabled. `EnableSmsNotifications` is set to the value specified in the command above.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsVirtualAppointmentsPolicy.md:48]:
```suggestion
Creates a new policy instance with the identity sms-disabled. `EnableSmsNotifications` is set to the value specified in the command.
```

File: teams/teams-ps/teams/New-CsTeamsVirtualAppointmentsPolicy.md [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsVirtualAppointmentsPolicy.md:129]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: teams/teams-ps/teams/Remove-CsTeamsVirtualAppointmentsPolicy.md [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsTeamsVirtualAppointmentsPolicy.md:98]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: teams/teams-ps/teams/Set-CsTeamsVirtualAppointmentsPolicy.md (hunk 2 above)
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsVirtualAppointmentsPolicy.md:24]:
```suggestion
Updates any of the properties of an existing instance of TeamsVirtualAppointmentsPolicy.
```

File: teams/teams-ps/teams/Set-CsTeamsVirtualAppointmentsPolicy.md [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsVirtualAppointmentsPolicy.md:115]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @emmanuelrocha001, kindly commit the suggested changes. Many thanks.

**Comment by emmanuelrocha001** [comment]
Anything else needed for approval @yogkumgit  @get-itips  @JohanFreelancer9 ?

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by emmanuelrocha001** [comment]
Looks like I don't have access to merge the PR, can this be completed @get-itips  @yogkumgit ?

**Comment by get-itips** [comment]
@tiburd

**Pull request merged**